alacritty.yml

```diff
@@ -23,8 +23,9 @@
 #window:
   # Window dimensions (changes require restart)
   #
-  # Specified in number of columns/lines, not pixels.
-  # If both are `0`, this setting is ignored.
+  # Number of lines/columns (not pixels) in the terminal. Allowed values are
+  # integers above 1 line and 2 columns; using `0` will follow the window
```

Comment:
Technically it's `>=`, so I would rephrase this to say the following, or something equivalent:

> Allowed values are integers greater than or equal to 1 line and 2 columns;

Perhaps, we must only describe the constraint for `columns` since `lines` is fully defined without this extra information. This could allow us to motivate the constraint a bit as well. In total:

> Number of columns/lines (not pixels) in the terminal. A value of `0` will follow the window manager's recommended size. Otherwise, `columns` must be at least 2 in order to support double-width characters.